app/Domain/Integrations/Repositories/EloquentIntegrationRepository.php

```diff
@@ -45,14 +45,14 @@ public function save(Integration $integration): void
 
     public function update(Integration $integration): void
     {
-        IntegrationModel::query()
-            ->findOrFail($integration->id->toString())
-            ->update([
-                'type' => $integration->type,
-                'name' => $integration->name,
-                'description' => $integration->description,
-                'subscription_id' => $integration->subscriptionId->toString(),
-                'status' => $integration->status,
+        $integrationModel = IntegrationModel::query()->findOrFail($integration->id->toString());
```

Comment:
Is there a special need for the local variable `$integrationModel`?

Reply:
I don't think so

Reply:
I see other instances in the code where it also defined first, to then call a method on it.
e.g.
```php
$integrationModel = IntegrationModel::query()->findOrFail($id->toString());
$integrationModel->activateWithOrganization($organizationId);
```

Reply:
But that line is to convert it a `IntegrationModel` and make PHPStan happy I think.